base/error.jl

```diff
@@ -91,28 +91,34 @@ function catch_backtrace()
     return _reformat_bt(bt[], bt2[])
 end
 
+struct ExceptionStack <: AbstractArray{Any,1}
+    stack
+end
+
 """
-    catch_stack(task=current_task(); [inclue_bt=true])
+    current_exceptions(task=current_task(); [inclue_bt=true])
 
 Get the stack of exceptions currently being handled. For nested catch blocks
 there may be more than one current exception in which case the most recently
-thrown exception is last in the stack. The stack is returned as a Vector of
-`(exception,backtrace)` pairs, or a Vector of exceptions if `include_bt` is
-false.
+thrown exception is last in the stack. The stack is returned as an
+`ExceptionStack` which is an AbstractVector of named tuples
+`(exception,backtrace)`. If `include_bt` is false, the backtrace in each pair
+will be set to `nothing`.
 
 Explicitly passing `task` will return the current exception stack on an
 arbitrary task. This is useful for inspecting tasks which have failed due to
 uncaught exceptions.
 """
-function catch_stack(task=current_task(); include_bt=true)
+function current_exceptions(task=current_task(); include_bt=true)
```

Comment:
Maybe `backtrace` instead of `include_bt`?

Reply:
Sure sounds good.

Reply:
Done.